### ISSUE TITLE
Improve synonymization of Technetium Tc-99m Albumin Aggregated

### ISSUE THREAD

**Issue by amykglen** [RTXteam/RTX#1046]
as @edeutsch requested, I traced a couple instances where the local fastNGD database (#729) 'misses' a concept but eUtils doesn't - this is the write-up for one example: NCIT:C87398 (Technetium Tc-99m Albumin Aggregated).

```
  - 2020-08-25 15:36:43.838157 DEBUG: Had to use eUtils to compute NGD between renal cell carcinoma (MONDO:0005086) and Technetium Tc-99m Albumin Aggregated (NCIT:C87398)(value is: 0.7492777150971099)
```

from kg2canonicalized:
n.id | n.name | n.equivalent_curies
-- | -- | --
"NCIT:C87398" | "Technetium Tc-99m Albumin Aggregated" | ["NCIT:C87398", "CHEMBL.COMPOUND:CHEMBL1201522"]

so in this case, we can see there's no MESH curie in the `equivalent_curies`, and I confirmed that neither of the equivalent nodes have `publications` listed in KG2, so it's not surprising fastNGD isn't aware of any PMIDs for this node.

but what is interesting is that there **is** a MESH node in KG2 named "Technetium Tc 99m Aggregated Albumin" (word order is slightly different):

n.id | n.name | n.equivalent_curies
-- | -- | --
"MESH:D013668" | "Technetium Tc 99m Aggregated Albumin" | ["UMLS:C0740185", "UMLS:C0087067", "UMLS:C0087068", "MESH:D013668"]

and there are definitely PubMed articles associated with MESH term D013668, so if `NodeSynonymizer` synonymized these two concepts, then the fastNGD system would no longer 'miss' this concept.

**Comment by amykglen** [comment]
well this is slightly better merged in the new synonymizer (#2003) - now a CHEMBL node has been added. though the MESH term is still on its own:

Cluster for NCIT:C87398 (CHEMBL.COMPOUND:CHEMBL1201522) has 2 nodes:

| id                            | category       | name                                 | in_SRI   | in_KG2pre   | is_cluster_rep   |
|:------------------------------|:---------------|:-------------------------------------|:---------|:------------|:-----------------|
| CHEMBL.COMPOUND:CHEMBL1201522 | ChemicalEntity | TECHNETIUM TC 99M ALBUMIN AGGREGATED | X        | X           | X                |
| NCIT:C87398                   | Drug           | Technetium Tc-99m Albumin Aggregated |          | X           |                  |

Cluster for MESH:D013668 has 1 nodes:

| id           | category   | name                                 | in_SRI   | in_KG2pre   | is_cluster_rep   |
|:-------------|:-----------|:-------------------------------------|:---------|:------------|:-----------------|
| MESH:D013668 | NamedThing | Technetium Tc 99m Aggregated Albumin |          | X           | X                |

kinda funny the SRI doesn't recognize this MESH term, since they do recognize most MESH

**Comment by edeutsch** [comment]
@amykglen can this be closed?

**Comment by dkoslicki** [comment]
And @amykglen , it looks like the NGD API endpoint handles this: https://arax.ci.transltr.io/api/arax/v1.4/ui/#/PubmedMeshNgd/pubmed_mesh_ngd
![image](https://github.com/RTXteam/RTX/assets/6362936/e49aed17-27ad-472b-b74b-e86ae2fdb42c)
Though I can't get this to work with DSL or a TRAPI query due to NGD-expansion not being a "thing"

**Comment by amykglen** [comment]
so I'm seeing three clusters in our latest synonymizer (KG2.9.2c) for Technetium Tc 99m albumin aggregated:
https://arax.ncats.io/beta/?term=MESH:D013668 (`Technetium Tc 99m Aggregated Albumin`)
https://arax.ncats.io/beta/?term=CHEMBL.COMPOUND:CHEMBL1201522 (`TECHNETIUM TC 99M ALBUMIN AGGREGATED`)
https://arax.ncats.io/beta/?term=KEGG.DRUG:D06023 (`Technetium Tc 99m albumin aggregated (USP)`)

the first two appear to arise from the fact that the SRI node normalizer assigns those identifiers to two such separate clusters:
https://nodenormalization-sri.renci.org/get_normalized_nodes?curie=MESH:D013668
https://nodenormalization-sri.renci.org/get_normalized_nodes?curie=CHEMBL.COMPOUND:CHEMBL1201522

but the third is for a KEGG node, which the SRI node normalizer doesn't currently support - so with that one, I think we could do better in our synonymization and assign it to one of the first two clusters.

@dkoslicki - that's really interesting that the NGD endpoint _does_ appear to map the name for the CHEMBL node to the main cluster for Technetium Tc-99m Albumin Aggregated, even though that CHEMBL node is still in a separate cluster in the synonymizer. I'm a bit perplexed as to how that would be happening.. I'm not familiar with that endpoint but I thought our NGD only uses the synonymizer to map concepts? but I suppose I'll take it!

so in summary - I wrote up an issue in the SRI NN repo about there being two clusters for Technetium Tc 99m albumin aggregated (https://github.com/TranslatorSRI/NodeNormalization/issues/280), but I think it's worth keeping this issue open for now due to the poor synonymization of the KEGG node (which is in our hands, since the SRI NN doesn't appear to support KEGG identifiers currently)

**Comment by dkoslicki** [comment]
Thanks for the sleuthing @amykglen ! I'll leave this open and tag as technical debt

**Comment by saramsey** [comment]
Fixed in KG2.10.2c, as shown below. Better late than never!

<img width="1658" height="1500" alt="Image" src="https://github.com/user-attachments/assets/7282d4ac-998d-4d37-9bda-51c3561254b8" />
